decentraland/unity-renderer
# feat: receive improved friend requests

## What does this PR change?

Adds a popup when you click over a received or sent friend request in the list.
Adds a notification when a friend request is accepted or received in the chat panel.

## How to test the changes?

Take in consideration that all the information displayed in `zone` is fake.

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/receive-friend-requests
2. Open the friend request list, you should see faked friend requests
3. Click any of the received requests
4. Reject the request. It should be removed from the list
5. Open another received request
6. Accept it
7. Click on any of the sent requests
8. Cancel it. It should be removed from the list
9. Go to: https://play.decentraland.org/?renderer-branch=feat/receive-friend-requests
10. The friend requests should be displayed normally and all the interactions should be working as usual

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
